static/app/views/projectDetail/projectScoreCards/projectVelocityScoreCard.spec.tsx

```diff
@@ -0,0 +1,108 @@
+import {OrganizationFixture} from 'sentry-fixture/organization';
+
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import ProjectVelocityScoreCard from './projectVelocityScoreCard';
+
+describe('ProjectDetail > ProjectVelocity', function () {
+  let allTimeDataEndpointMock: jest.Mock;
+  let currentDataEndpointMock: jest.Mock;
+  let previousDataEndpointMock: jest.Mock;
+  const organization = OrganizationFixture();
+
+  const selection = {
+    projects: [1],
+    environments: [],
+    datetime: {
+      start: null,
+      end: null,
+      period: '14d',
+      utc: null,
+    },
+  };
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  afterEach(function () {
+    MockApiClient.clearMockResponses();
+  });
+
+  it('renders release count', async function () {
+    previousDataEndpointMock = MockApiClient.addMockResponse({
```

Comment:
since these aren't set in the beforeEach i don't think it makes sense to declare them in a shared way
```suggestion
    const previousDataEndpointMock = MockApiClient.addMockResponse({
```